vue-router-page-guard/vue-router-page-guard.md

```diff
@@ -0,0 +1,29 @@
+# Vue Router Page Guard
+In other words, protecting a page by only allowing users with sufficient permissions to use the route to it
+
+[Full Vue Documentation](https://router.vuejs.org/guide/advanced/navigation-guards.html#in-component-guards). It's not a very complex topic, but here is a short summary:
+
+### Per-Route Guard
+Adding a guard on a specific route. Say, protecting a route that goes to some sort of admin panel page where an adminstrator can review and change data (e.g. users).
+
+```javascript
+const router = new VueRouter({
+  routes: [
+    {
+      path: '/foo',
+      component: Foo,
+      beforeEnter: (to, from, next) => {
+        // ...
+      }
+    }
+  ]
+})
+```
+
+Where **to** is a Route to be routed to, **from** is the Route being navigated away from, and **next** is a function that resolves the hook depending on what arguments it is given.
+
+No arguments in **next**, simply **next()**, resolves the hook and allows the navigation to happen.
+
+**next(false)** will block the route and re-route back to the **from** route.
```

Comment:
Looks great the one thing that I would add is how this works in tandem with dynamically shown components. The links to these pages should all be behind access controls.